sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -492,6 +560,7 @@ export interface BlobSetTierOptions {
    * @memberof BlobSetTierOptions
    */
   leaseAccessConditions?: Models.LeaseAccessConditions;
+  rehydratePriority?: Models.RehydratePriority;
```

Comment:
Minor: add the @type and @memberof info, I find these are added for track 2, would better to make this consistent if this is convention for track 2.

Reply:
Added